lib/combinat.gi

```diff
@@ -2713,11 +2713,10 @@ end);
 ##
 #F  Bernoulli( <n> )  . . . . . . . . . . . . value of the Bernoulli sequence
 ##
-BindGlobal( "Bernoulli2",
-    [-1/2,1/6,0,-1/30,0,1/42,0,-1/30,0,5/66,0,-691/2730,0,7/6] );
```

Comment:
Note: I deliberately dropped the initial values: they are super cheap to recompute, and afterwards are cached anyway. Also, I initially did provide these defaults, but found it harder to understand the cod with it (as the list of defaults comes right after the function being memoized).